projects/plugins/crm/webpack.config.js

```diff
@@ -9,6 +9,7 @@ const welcomeZBSCSSPattern = '**/css/welcome-to-zbs/*.css';
 
 const alwaysIgnoredFiles = [
 	'**/js/**/*.min.js',
+	'**/src/js/**',
```

Comment:
We have to ignore `src/js` from our "legacy JS" build to prevent errors due to requiring other dependencies.